air/src/ir/bus.rs

```diff
@@ -14,6 +15,31 @@ pub struct Bus {
     pub first: NodeIndex,
     /// The final state of the bus
     pub last: NodeIndex,
```

Comment:
Does using `NodeIndex` here make sense any more? I think currently boundary conditions of a bus are limited to:

- Empty bus (using the `null` keyword).
- A bus initialized to some public input (of type table).

I don't think either of these needs to be in the algebraic graph, right?